api/security.py

```diff
@@ -7,6 +7,7 @@ class PhoneLockTimeResponse(GenericResponse):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.phoneLockTime = self.response.body["phoneLockTime"]
+        self.timeLeftToNextAttempt = self.response.body["timeLeftToNextAttempt"]
```

Comment:
please add documentation for related api :) it will save quite a bit of time for someone trying to wrap his head around it

Reply:
I've already updated the documentation for this endpoint API:
https://appnroll.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=1299415622&selectedPageVersions=15&selectedPageVersions=13